### DISCUSSION TITLE
I have made a website for this project~

### DISCUSSION THREAD

**Discussion by Renzehua1998** [ayaka14732/ChatGPTAPIFree#26] [Ideas]
repo：https://githubfast.com/Renzehua1998/Free-GPT-UI
online testing： https://renzehua1998.github.io/Free-GPT-UI
Welcome to test and make suggestions~

**Comment by ayaka14732** [comment]
This is nice! Actually we have an official frontend [Free ChatGPT](https://github.com/ztjhz/FreeChatGPT). Besides, it is worth noticing that we are soon running out of fee. In this case, we encourage developers to setup their own API endpoints.

**Reply by Renzehua1998** [reply]
Thank you! This frontend is magnificent!
